CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# Upcoming
+
+* Empty functions `function () {}` replaced by `angular.noop`
```

Comment:
@wytrych Those don't affect the distribution so they're not required to be listed in the changelog; I don't think they're useful to read. Users of this package don't need to know about internal changes that don't affect the code they're using.

If they want to see those changes, the commit history will show them =)